backend/lib/extensions/json_map_extension.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:backend/utils/type_utils.dart';
+import 'package:googleapis/firestore/v1.dart';
+
+extension JsonMapExtension on JsonMap {
```

Comment:
What do you think of moving JsonMapExtension to "backend/lib/utils/type_utils.dart " same place where we defined JsonMap type we will have the befits of always being able to use the "toDocument" without the need to import two files?

Reply:
Definitely see your point, extensions are a pain with importing hey. Sounds good!

Reply:
Done 👍

Reply:
Oh sorry did I jump the gun? I didn't check if you were done with your review when I requested re-review.  Anyway I'm off to bed now but back on in the morning... talk to you then 🙂